CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-## 2017-02-14
+##[2.6.3] - 2017-02-16
```

Comment:
I think it makes sense to call it `3.0` or `2.7` at least. point releases should not have breaking changes and following SemVer it should be major bump.